cernopendata_client/validator.py

```diff
@@ -41,6 +41,11 @@ def validate_range(range=None, count=None):
     Return True if range lies in total files count,
     raises click.BadParameter exception otherwise.
     """
+    if len(range.split("-")) != 2:
```

Comment:
Hmm, currently with `master` version the single digit input also works fine; I mean that this:

```
 $ cernopendata-client download-files --recid 5500 --filter-range 1
```

is the same as this:

```
 $ cernopendata-client download-files --recid 5500 --filter-range 1-1
```

Which is perhaps nice to keep?

However, we can also require to always have A-B as you mention, so that the notion of "range" is ultra clear :+1:

Reply:
I think I will prefer keeping the start and end in the `range` option else users can always use the `name` option if they want to download a specific file.

But I am open to adding only one index in range.